src/auth/session/storage/custom.ts

```diff
@@ -5,20 +5,32 @@ import * as ShopifyErrors from '../../../error';
 
 export class CustomSessionStorage implements SessionStorage {
   constructor(
-    readonly storeCallback: (session: SessionInterface) => Promise<boolean>,
-    readonly loadCallback: (
+    readonly storeSessionCallback: (
+      session: SessionInterface,
+    ) => Promise<boolean>,
+    readonly loadSessionCallback: (
       id: string,
     ) => Promise<SessionInterface | {[key: string]: unknown} | undefined>,
-    readonly deleteCallback: (id: string) => Promise<boolean>,
+    readonly deleteSessionCallback: (id: string) => Promise<boolean>,
+    readonly deleteSessionsCallback?: (
+      id: string[],
+    ) => Promise<boolean> | undefined,
```

Comment:
I think we can _not_ return `undefined` here - if the function is defined, we should just force the response type to be a `Promise<boolean>`.

```suggestion
    ) => Promise<boolean>,
```

Reply:
👍🏻